src/UnitTests/UnitTests.csproj

```diff
@@ -1,13 +1,13 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
-    <TargetFrameworks>netcoreapp2.0</TargetFrameworks>
+    <TargetFrameworks>netcoreapp3.0</TargetFrameworks>
```

Comment:
`netcoreapp3.1` is probably better, since `netcoreapp3.0` support ended March 3 2020